hr_infraction/models/hr_employee.py

```diff
@@ -0,0 +1,13 @@
+from openerp import models, fields
+
+
+class HrEmployee(models.Model):
+
+    _name = 'hr.employee'
```

Comment:
Please see http://odoo-new-api-guide-line.readthedocs.org/en/latest/environment.html about inheritance, it's odoo standard that you inherit not to use the _name, only in certain situation, when you want to create a new model from old one, example project.project with analytic.account...